packages/mobile/src/screens/app-drawer-screen/left-nav-drawer/LeftNavDrawer.tsx

```diff
@@ -35,25 +34,25 @@ const WrappedLeftNavDrawer = () => {
   const { navItems } = useNavConfig()
 
   return (
-    <DrawerContentScrollView>
-      <AccountDetails />
-      <VanityMetrics />
-      {navItems.map((item) => (
-        <LeftNavLink
-          key={item.label}
-          icon={item.icon}
-          label={item.label}
-          to={item.to}
-          params={item.params}
-          onPress={item.onPress}
-          showNotificationBubble={item.showNotificationBubble}
-        >
-          {item.rightIcon}
-        </LeftNavLink>
-      ))}
-      <Flex pt='5xl' ph='l'>
-        <IconAudiusLogo color='subdued' />
+    <Flex h='95%' pt='unit16' justifyContent='space-between'>
```

Comment:
can we wrap both in a flex that has some better props? down to huddle if useful

Reply:
wdym exactly by better props?

Reply:
like justify content or something

Reply:
what's the reason for `95%` instead of `100%`?